wgsl/index.bs

```diff
@@ -6381,14 +6381,18 @@ When an identifier is used as a [=syntax/callable=] item, it is one of:
 <div class='syntax' noexport='true'>
   <dfn for=syntax>lhs_expression</dfn> :
 
-    | ( [=syntax/star=] | [=syntax/and=] ) * [=syntax/core_lhs_expression=] [=syntax/component_or_swizzle_specifier=] ?
+    | [=syntax/core_lhs_expression=] [=syntax/component_or_swizzle_specifier=] ?
 </div>
 <div class='syntax' noexport='true'>
   <dfn for=syntax>core_lhs_expression</dfn> :
 
     | [=syntax/ident=]
 
     | [=syntax/paren_left=] [=syntax/lhs_expression=] [=syntax/paren_right=]
+
```

Comment:
This makes * and & higher precedence than [ ] and .member

The WGSL committee rejected this, because
- want it more like C
- want LHS parsing to have the same precedence as RHS parsing.